### ISSUE TITLE
Login page

### ISSUE THREAD

**Issue by iandunn** [WordPress/wporg-news-2021#61]
https://www.figma.com/file/O9I8Jl7kaUMTX39LlRGbGO/News-Page?node-id=8235%3A33015

**Comment by dd32** [comment]
Just noting that I'll look into this - although that it's not a small amount of work.

We'll have to remove `wp-login.php` from the picture for a lot of the actions it handles (I don't believe we'll simply be able to re-style `wp-login.php` as-is), which will require a lot more work than may be anticipated.

**Comment by ryelle** [comment]
Out of curiosity, can someone screenshot whatever the figma link is supposed to be here? I just keep getting dropped into the main page & don't see anything login-y.

**Comment by iandunn** [comment]
> We'll have to remove wp-login.php 

Is that really worth it? Maybe we can adjust the design to fit the constraints instead.

>  just keep getting dropped into the main page & don't see anything login-y.

It still works for me, but it's on the `i7` page. @beafialho , is that finalized, or something we should hold off on?

**Comment by beafialho** [comment]
> It still works for me, but it's on the i7 page. @beafialho , is that finalized, or something we should hold off on?

Yeah, I think it's wisest to hold off on the login page. In the future we may have to rethink that top bar and it could change.

**Comment by dd32** [comment]
Yeah, have to be logged into Figma (any account will do) for the above link to work.

There's not a lot of specifics included in the figma design, no errors styles, not all screens covered but there's enough to make it match the news design closer, and honestly there aren't too many different designs between the screens anyway.

![image](https://user-images.githubusercontent.com/767313/151292060-926ee216-7ddb-4849-8b30-5ce92b1c2983.png)

> > We'll have to remove wp-login.php
>
> Is that really worth it? Maybe we can adjust the design to fit the constraints instead.

No, no it's not. But adjusting the design won't work here as the wp-login design constraint is that wp-login is not designed to be adjusted (In part, that's why the existing one matches the wp-login design so much)

There's a few workarounds though, as styling wp-login is done by plugins, there's also a few more horrible ways that I prefer, such as using output buffering to discard the entire wp-login forced `<html>` header.

**Comment by dd32** [comment]
Based on the above, shifting out of the milestone. 

I've drafted a PR over at https://github.com/WordPress/wordpress.org/pull/53 with the small amount of what I hacked together as a learning exercise. 
(See PR for WIP images)

Personally, I much prefer the cleaner design which is why I went for it and started without confirming if it was finalised (given that it wasn't in the final deck).

I feel like it's a worthy enhancement to the design for a often-used page (Everyone see's it at least once every 2 weeks).

> In the future we may have to rethink that top bar and it could change.

The top bar is actually quite a distraction IMO, however, if the navigational items were ditched from it, it's not that bad (See https://github.com/WordPress/wordpress.org/pull/53#issuecomment-1025432755 )
